test/cypress/integration/22-collective.edit.test.js

```diff
@@ -56,6 +57,12 @@ describe('edit collective', () => {
     cy.getByDataCy('create-collective-mini-form').should('not.exist'); // Wait for form to be submitted
     cy.getByDataCy('confirmation-modal-continue').click();
     cy.get('[data-cy="member-1"] [data-cy="member-pending-tag"]').should('exist');
+    cy.getInbox().should('have.length', 2);
+
+    // Re-send the invitation email
+    cy.getByDataCy('resend-invite-btn').should('exist').first().click({ force: true });
+    cy.wait(200); // Wait for email
+    cy.getInbox().should('have.length', 3);
```

Comment:
Nice :+1:

Reply:
I love how straightforward it was to design this test